Firmware/MotorControl/controller.hpp

```diff
@@ -95,8 +95,6 @@ class Controller : public ODriveIntf::ControllerIntf {
 
     bool anticogging_valid_ = false;
 
-    // custom setters
-    void set_input_pos(float value) { input_pos_ = value; input_pos_updated(); }
```

Comment:
Did this get moved somewhere?  Just to check, the trap_traj planner needs input_pos_updated() called so that it can plan a new trajectory.

Reply:
Good catch, I added it back.